### PR TITLE
fix: Resolve issue  #1564

### PR DESCRIPTION
**Pull request by GeraAlcantara**
## Description

I've store CSS varibles in an object
in the _layout.svelte
And  added a varible named theme to store the iteration key, value  and returning -- key:value and join;
--key:value;
to converted to a proper CSS 

and I have ass the varible to the <main> block 

Also add a comment 
asking for more  info 

**Is this related to any open issue(s)?**

fixes #{1564}

**Is there anything you need help with to get this PR merged?**

Advice if the root varible have to be remove from mystyles.scss.

And where on the _layout.svelte you need the styles apply 

_..._

## Checklist

Don't worry if you haven't done everything on this checklist. Simply create your PR, and make sure everything is done later,
or ask for help if you don't know how to do it.

- [ ] The CI is passing
- [x ] I tested my implementation manually
- [ ] I wrote new tests to test my code, or updated existing tests to test my changes

### CODE REVIEW

**Comment by kantord** [comment]
Hi, nice MR! lemme see :eyes:

File: apps/web/src/mystyles.scss [hunk 1]
```diff
@@ -58,6 +58,8 @@
 
 @import "~bulmaswatch/lumen/overrides.scss";
 
+/* TODO: Plobably after PR this root elements will be removed */
+
 :root {
     --spacing-small: 8px;
     --spacing-medium: 12px;
```
Comment (kantord) [inline apps/web/src/mystyles.scss:65]: I think everything that you move to the CSS should be removed from here, that's the only way we can know the other stuff _really_ works

File: apps/web/src/routes/_layout.svelte [hunk 2]
```diff
@@ -41,7 +85,7 @@
 {#if isBrowser() !== true}
   <div class="pageloader is-active"><span class="title">LibreLingo</span></div>
 {/if}
-
-<main>
+<!-- Plase advice where you need to apply the styles to body or here in main  -->
```
Comment (kantord) [inline apps/web/src/routes/_layout.svelte:88]: I would say this is the best place if it works !

Comment (GeraAlcantara) [reply apps/web/src/routes/_layout.svelte:88]: ok got it 👍

File: apps/web/src/mystyles.scss (hunk 1 above)
Comment (GeraAlcantara) [reply apps/web/src/mystyles.scss:65]: ok

**Comment by GeraAlcantara** [comment]
Changes apply can you help me review it one more time @kantord 
Thank you

**Comment by kantord** [comment]
Amazing @GeraAlcantara 💯

looks good to me! Let me know if you want to contribute more and need help choosing an issue or implementing it

@allcontributors please add @GeraAlcantara for code

**Pull request merged**